content/dev/dev-guide.md

```diff
@@ -102,79 +56,74 @@ A desk is a series of numbered commits, the most recent of which represents the
 For building applications to distribute on Urbit, we'll want to work on a new desk.
 
 #### Desk basics
+##### Setting up a new desk
+###### With Citadel
+The Citadel app allows you to create and manipulate your desks purely from the dojo. It provides a repl and a split screen front end that works with the docs app.
+ >  (You might be reading this on urbit already - skip the following if so)
+- Install on the livenet from `~mipber/citadel`, either via the UI or by typing `|install ~mipber %citadel` into the dojo. Once it finishes, type `|link %citadel-cli` to switch to a repl with custom commands reached through the `;` prefix. These allow you to create desks from examples or simply use another desk as a starting point.
+- Install `~pocwet/docs` to read this guide natively in urbit.
+- In the `citadel` repl:
+      ```
+         ;desk hello ~ hel
+      ```
 
-##### Mounting and unmounting Desks
-
- - Mounting a desk allows you to add files to it (it creates a directory you can see on the host filesystem). Run these commands in the dojo.
-   - ```bash
-     |mount %base
-     ```
- - You can similarly unmount a desk
-   - ```bash
-     |unmount %base
-     ```
+   This will create a desk `%hello` from the `hel` template, which contains all the dependencies we need to create a simple 'hello world' app (along with a sample gall agent).
+- In the `dojo`:
+    ```
+    |mount %hello
+    ```
 
-##### Setting up a new desk
+This will mount the files to a directory on the host filesystem so they can be edited using your preferred editor.
 
+###### Manual Steps
```

Comment:
I think we should commit to one thing here, and if that's citadel we should make it citadel (and make sure it works).

This has the benefit of keeping things easier to support and at a high quality bar and reducing the steps in the getting started guide.